src/NServiceBus.Metrics.ServiceControl/ServiceControlReporting/QueueLength/QueueLengthReport.cs

```diff
@@ -0,0 +1,7 @@
+﻿using System.Collections.Generic;
+
+class QueueLengthReport
+{
+    public long TimeStamp { get; set; }
+    public Dictionary<string, long?> Queues { get; } = new Dictionary<string, long?>();
```

Comment:
It's accessed from different threads. Should we make it a `ConcurrentDictionary<string,long?>`?